framework/client/flash-context.js

```
@@ -0,0 +1,58 @@
+      addTimeouts(messages);
+      setFlashMessages((prevMessages) => {
+        return [...prevMessages, ...messages];
+      });
+    };
+
+    window.__cosmos_rsc = { ...window.__cosmos_rsc, flash };
+
+    addTimeouts(initialState)
+
+    return () => {
+      delete window.__cosmos_rsc.flash;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)